conda/meta.yaml

```diff
@@ -24,6 +24,7 @@ test:
     - ts-conda-build =0.3
     - ts-tcpip
     - ts-utils
+    - pytest
```

Comment:
`pytest` is part of `ts-conda-build`... you can remove it altogether from here

https://github.com/lsst-ts/ts_recipes/blob/fa6ac2c877a3a2966ea8dcaa3c39104a30500940/ts_conda_build/meta.yaml#L16